### PR TITLE
M3-4687 Primary Nav and Icon updates

### PR DESCRIPTION
**Pull request by johnwcallahan**
## Description

- Icon Updates (used in Primary Nav, Activity Landing, Search Landing, etc.)
- Dark Theme improvements Primary Nav
- Search Bar enhancements:
  - Replace red "offline" icon with grey (feedback from Chris)
  - Bolden entity label instead of description (feedback from Chris)
- Main Content max-width at 1280px (our large breakpoint) instead of 1100px
- Edited copy in Create menu (pending approval/edits)

I started out trying to make this work for CMR and non-CMR, but things got messy with the icons. So, this is the point of no return (it'll break non-CMR). I temporarily set the CMR flag to `false` (hardcoded) in this PR. 

Todo: 

- [ ] final icons
- [ ] approval of newly edited copy
- [x] dark theme

Screen Shot 2020-12-04 at 4 48 22 PM

### CODE REVIEW

File: packages/manager/src/themeFactory.ts [hunk 1]
```diff
@@ -1567,6 +1568,17 @@ export default (options: ThemeOptions & ThemeOverrides) =>
       themeDefaults({
         spacingOverride: options.spacingOverride
       }),
-      options
+      {
+        breakpoints: {
```
Comment (johnwcallahan) [inline packages/manager/src/themeFactory.ts:1572]: These didn't end up changing, but I thought I'd leave them here so we have them in writing in our codebase.

**Review by Jskobos — APPROVED** [review]
Looks ok (and I think we should get this in and handle pending tweaks later). Documenting a few things:

1. Mobile has some weird scrolling (i think this is being handled elsewhere but it's different than on develop so I'm screenshotting for posterity:

Screen Shot 2020-12-07 at 1 16 35 PM

2. There's no gutter at all on smaller viewports, which looks odd to my eye for our banners:

Screen Shot 2020-12-07 at 1 08 43 PM

3. The navbar toggle is sending the correct request to /preferences but it's not sticking somehow

**Comment by Jskobos** [comment]
the busy status in the search bar results looks weird (and keeps spinning):

Screen Shot 2020-12-07 at 1 22 47 PM

**Review by tiffwong — APPROVED** [review]
Other than the things @Jskobos mentioned, the `cmrSpacing` breakpoint should start at 1480px (1280px + left nav width) instead or it'll be flush to the edge

Screen Shot 2020-12-07 at 2 36 52 PM

**Pull request merged**